projects/extension/src/background/ConnectionManager.ts

```diff
@@ -215,13 +232,38 @@ export class ConnectionManager
     this.#client = undefined
   }
 
+  /**
+   * init Runs on onStartup and onInstalled of extension and
+   * connects the default relayChains to smoldot
```

Comment:
You shouldn't document when a function is called, that's the problem of the user of the code. `init` is here to initialize the state machine, and that's it. That's all there is to document.